velox/exec/RowContainer.h

```diff
@@ -1466,14 +1498,10 @@ class RowContainer {
       char* row,
       int32_t columnIndex);
 
-  // Light weight aggregated column stats does not support row erasures. This
+  // Min/max column stats do not support row erasures. This
   // method is called whenever a row is erased.
-  void invalidateColumnStats();
-
-  // Updates the specific column's columnHasNulls_ flag, if 'hasNulls' is true.
-  // columnHasNulls_ flag is false by default.
-  inline void updateColumnHasNulls(int32_t columnIndex, bool hasNulls) {
-    columnHasNulls_[columnIndex] = columnHasNulls_[columnIndex] || hasNulls;
+  void invalidateMinMaxColumnStats() {
```

Comment:
When we use this?

Reply:
When we have eraseRows or setAllNull

Reply:
Maybe some additional ask. And maybe we can discuss this and potentially have a followup: Fixed column size stats can immune from erasure. Do we want to indicate in the column stats if this column is fixed/variable length column? We just need to set the stats size at construction time and skip updating them. Might save some cycles.

Reply:
Good idea. I'll make a follow up task for this.